source/ShipInfoDisplay.cpp

```diff
@@ -44,11 +44,11 @@ ShipInfoDisplay::ShipInfoDisplay(const Ship &ship, const PlayerInfo &player, boo
 
 
 
-// Call this every time the ship changes.
-void ShipInfoDisplay::Update(const Ship &ship, const PlayerInfo &player, bool descriptionCollapsed)
+// Call this every time the ship changes. Sale is true for the shipyard and outfitter panels only.
```

Comment:
Shoudnt  we explain why its only true in those cases?
Actually renaming it from sale to advancedPanel should clarify that?